xcube/webapi/datasets/routes.py

```diff
@@ -19,20 +19,119 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+from xcube.server.api import ApiError
 from xcube.server.api import ApiHandler
-
 from .api import api
 from .context import DatasetsContext
+from .controllers import find_dataset_places
+from .controllers import get_color_bars
+from .controllers import get_dataset_place_group
+from .controllers import get_datasets
+from ...util.assertions import assert_true
 
 
-# noinspection PyAbstractClass
-@api.route("/datasets")
+@api.route('/datasets')
 class DatasetsHandler(ApiHandler[DatasetsContext]):
+    """List the published datasets."""
+
     @api.operation(
-        operationId='getDatasets',
-        summary='Get all the datasets published by this service.'
```

Comment:
Why remove the summary?

Reply:
Have a look at the generated page. This is how Swagger works.

Reply:
It displays the summary just next to the route path which is very handy.

Reply:
Ok, but why keep the summary e.g. in line 86?

Reply:
Aaah, _remove_! I read _add_. I'll fix it.

Reply:
Fixed.